src/components/schemas/invitation.yaml

```diff
@@ -139,3 +139,12 @@ invitation_in:
       example: [ 'delegatee@example.com' ]
       description: |
         List of emails of employees that are delegatees for this employee.
+
+    category_names:
```

Comment:
lgtm but can we add this section next to the projects / cost centers since they all are almost related.

Reply:
changed order